### PR TITLE
chore: publish docker image on release

### PR DESCRIPTION
**Pull request by ovesh**

### CODE REVIEW

**Comment by ovesh** [comment]
Not tested yet, waiting on a docker hub user.

**Comment by s22chan** [comment]
having trouble figuring out which python version is used

**Comment by ovesh** [comment]
> having trouble figuring out which python version is used

Hmm you're right, it's kind of arbitrary https://github.com/deepgenomics/GenomeKit/pull/112/files#diff-4aee7e10b78db377004a7ac425c4ed6e265bbdfbb6fdf64141ade09102fa75c6R61

I'll fix it so it's explicitly py3.10

File: .github/workflows/dockerize.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+      - name: Build and push
+        uses: docker/build-push-action@v6
+        with:
+          context: ~/micromamba/envs/build/envs
+          file: Dockerfile
+          push: true
+          tags:
+            - deepgenomicsinc/genomekit:latest
+            - deepgenomicsinc/genomekit:${{ steps.create-gk-env.outputs.PGK_VERSION }}
```
Comment (s22chan) [inline .github/workflows/dockerize.yaml:91]: can we change the label so denotes the python version

**Review by s22chan — APPROVED** [review]

**Pull request merged**